filebeat/module/elasticsearch/audit/test/test-access.log-expected.json

```diff
@@ -1,6 +1,5 @@
 [
     {
-        "@timestamp": "2018-06-19T05:16:15.549Z",
```

Comment:
@ycombinator Do you know why these timestamps are removed?

Reply:
Is the removal of these lines needed to get CI green? If not I'd keep them out of this PR.

Reply:
It is what happens when I regenerate the files. For now will remove all the module changes and skip the module tests with a follow up PR.

Reply:
Those `@timestamp`s should not be removed. I will regenerate the files and make sure they are there.